Sources/ScipioKit/Producer/PIF/ToolchainGenerator.swift

```diff
@@ -44,11 +44,11 @@ struct ToolchainGenerator {
         // Compute common arguments for clang and swift.
         var extraCCFlags: [String] = []
         var extraSwiftCFlags: [String] = []
-        let sdkPaths = try SwiftSDK.sdkPlatformFrameworkPaths(environment: [:])
-        extraCCFlags += ["-F", sdkPaths.fwk.pathString]
-        extraSwiftCFlags += ["-F", sdkPaths.fwk.pathString]
-        extraSwiftCFlags += ["-I", sdkPaths.lib.pathString]
-        extraSwiftCFlags += ["-L", sdkPaths.lib.pathString]
+        let macosSDKPlatformPaths = try await resolveSDKPlatformFrameworkPaths()
+        extraCCFlags += ["-F", macosSDKPlatformPaths.frameworkPath.pathString]
+        extraSwiftCFlags += ["-F", macosSDKPlatformPaths.frameworkPath.pathString]
+        extraSwiftCFlags += ["-I", macosSDKPlatformPaths.frameworkPath.pathString]
+        extraSwiftCFlags += ["-L", macosSDKPlatformPaths.frameworkPath.pathString]
```

Comment:
```suggestion
        extraSwiftCFlags += ["-I", macosSDKPlatformPaths.libPath.pathString]
        extraSwiftCFlags += ["-L", macosSDKPlatformPaths.libPath.pathString]
```

Reply:
@ikesyo thank you for spotting this!

Reply:
@dmhts Could you update this?

Reply:
Done!

Reply:
> Did you catch it by running the integration test?

Just by skimming 👀

Reply:
I see, but you're running integration tests before merging PRs, right? I believe this particular issue would've been caught by the test.

Reply:
We are not sure how the settings affect actual builds thus not sure how to test if it works correctly unfortunately (at least for now) 🙃